app/app.ts

```diff
@@ -1,2 +1,11 @@
-﻿import * as app from 'application';
-app.start({ moduleName: "main-page" });
+﻿import * as application from 'application';
+import {TNSFontIcon, fonticon} from 'nativescript-fonticon';
+
+TNSFontIcon.debug = true;
+TNSFontIcon.paths = {
+  'fa': 'font-awesome.css'
+};
+TNSFontIcon.loadCss();
+
+application.resources['fonticon'] = fonticon;
```

Comment:
Correct me if I’m wrong, but the 6-7 lines of code above will need to be included in every app that uses this theme for it to function correctly, right?

If so I’d recommend that we create a TypeScript API that we ship with this theme that can take care of this common setup. Something like...

``` javascript
import { setup } from "nativescript-theme-core";
setup();
```

That approach would give us the added ability to handle other JavaScript-y things we need to use the theme.